mkdocs/docs/reference/tasks/common.md

```diff
@@ -8,7 +8,7 @@ template: main.html
 
 ### Overview
 
-The `common/bash` task executes a bash script. The script can be written to use placeholders that are [dynamically substituted at runtime](../interpolation). For example, the `common/bash` task can be used as part of a finish action to promote the winning version at the end of an experiment.
+The `common/bash` task executes a bash script. The script can be written to use placeholders that are [dynamically substituted at runtime](../../tasks#dynamic-variable-substitution). For example, the `common/bash` task can be used as part of a finish action to promote the winning version at the end of an experiment.
```

Comment:
Please use link in the following format `../tasks.md#dynamic-variable-substitution` (notice .md) and the relative path.

I am currently changing all links to be along the above lines (as documented by mkdocs).